networkServer/src/main/java/org/pesc/cds/model/TranscriptRequestBuilder.java

```diff
@@ -0,0 +1,589 @@
+               </Request>
+         </ns1:TranscriptRequest>
+         */
+        RequestType request = academicRecordObjectFactory.createRequestType();
+        transcriptRequest.getRequests().add(request);
+        request.setCreatedDateTime(new Date());
+        RequestedStudentType requestedStudent = academicRecordObjectFactory.createRequestedStudentType();
+        request.setRequestedStudent(requestedStudent);
+        requestedStudent.setReleaseAuthorizedIndicator(studentRelease);
+        requestedStudent.setReleaseAuthorizedMethod(studentReleasedMethod);
+        PersonType person = academicRecordObjectFactory.createPersonType();
+        requestedStudent.setPerson(person);
+        if(StringUtils.isNotBlank(studentPartialSsn)) {
+            person.setPartialSSN(studentPartialSsn);
+        }
+        if(studentBirthDate!=null) {
+            BirthType birth = coreMainObjectFactory.createBirthType();
+            person.setBirth(birth);
+            birth.setBirthDate(studentBirthDate);
+        }
+        NameType name = coreMainObjectFactory.createNameType();
+        person.setName(name);
+        name.setFirstName(studentFirstName);
+        name.getMiddleNames().addAll(studentMiddleNames);
+        name.setLastName(studentLastName);
+        if(studentEmail!=null) {
+            ContactsType contact = academicRecordObjectFactory.createContactsType();
+            person.getContacts().add(contact);
+            EmailType email = academicRecordObjectFactory.createEmailType();
+            contact.getEmails().add(email);
+            email.setEmailAddress(studentEmail);
+        }
+        AttendanceType attendance = academicRecordObjectFactory.createAttendanceType();
+        requestedStudent.getAttendances().add(attendance);
+        attendance.setCurrentEnrollmentIndicator(studentCurrentlyEnrolled);
+        SchoolType school = academicRecordObjectFactory.createSchoolType();
+        attendance.setSchool(school);
+        school.setOrganizationName(studentSchoolName);
+        setSchoolCode(studentSchoolCodes, school);
+        ValidationResponse validationResponse = TranscriptRequestValidator.validateTranscriptRequestRequiredContent(transcriptRequest);
+        if (validationResponse.getSeverity() == SeverityCodeType.FATAL_ERROR || validationResponse.getSeverity() == SeverityCodeType.ERROR) {
+            SyntaxErrorType error = functionalAcknowledgmentObjectFactory.createSyntaxErrorType();
+            error.setErrorMessage("Required content missing for documentId: "+documentID);
+            error.setSeverityCode(SeverityCodeType.ERROR);
+            validationResponse.addErrorToTop(error);//put on top
+            log.error(validationResponse.toString());
+            throw new IllegalStateException(validationResponse.toString());
+        } else if(validationResponse.getErrors().size()>0){
+            SyntaxErrorType warnning = functionalAcknowledgmentObjectFactory.createSyntaxErrorType();
```

Comment:
You were feeling the love with the "n" key on this one. No big deal, just thought I'd throw it out there. :)